Makefile

```diff
@@ -662,6 +662,13 @@ test-e2e: ## Run "docker-build" and "docker-push" rules then run e2e tests.
 	$(MAKE) docker-build docker-push \
 	test-e2e-run
 
+.PHONY: test-e2e-existing
+test-e2e-existing:
```

Comment:
it's not immediately clear what "existing"  means in this context, could be an existing cluster for example, consider naming this and the below target more explicitly